### ISSUE TITLE
Training on many datasets

### ISSUE THREAD

**Issue by naayem** [MIC-DKFZ/nnUNet#1225]
Hello, Many thanks for this great framework. 
I am working on a master thesis using nnU-net. If you prefer I post my question in an existing issue like fine tuning please let me know.

Is there a good strategy to use different datasets to train one model? What worry me the most is the Ruled based parameters inferred from the Distribution of Spacings. Should I combine all data in one Directory as a new dataset or will it just confuse the plan and preprocess even more? Or maybe just use the plan and preprocess for one target dataset and pre-train on the others?
Example: Let's get better results on the AMOS test set by using AbdomenCT, BTCV and AMOS training set.

@FabianIsensee Should I just follow your indications here #1108  ? So I Would just iterate this fine tuning on the many datasets and train with the according plans.

```
nnUNet_plan_and_preprocess -t T2  # run planning and preprocessing for target dataset
nnUNet_plan_and_preprocess -t T1 -overwrite_plans PLANS_FROM_T2 -overwrite_plans_identifier my_t2_plans

nnUNet_train 3d_fullres nnUNetTrainerV2 T1 all -p my_t2_plans # run the pretraining
nnUNet_train 3d_fullres nnUNetTrainerV2 T2 FOLD_ID -pretrained_weights PATH_TO_MODEL_FINAL_CHECKPOINT # run the fine tuning
```
